catalog-rest-service/src/main/resources/ui/src/components/common/entityPageInfo/EntityPageInfo.tsx

```diff
@@ -81,47 +96,90 @@ const EntityPageInfo = ({
+                    className="tw-bg-gray-200"
+                    key={index}
+                    tag={`#${tag.tagFQN}`}
+                  />
+                ))}
 
-            {tags.slice(LIST_SIZE).length > 0 && (
-              <PopOver
-                className="tw-py-1"
-                html={
-                  <>
-                    {tags.slice(LIST_SIZE).map((tag, index) => (
-                      <Tags
-                        className="tw-bg-gray-200 tw-px-2"
-                        key={index}
-                        tag={`#${tag.tagFQN}`}
-                      />
-                    ))}
-                  </>
-                }
-                position="bottom"
-                theme="light"
-                trigger="click">
-                <span className="tw-cursor-pointer tw-text-xs link-text">
-                  View more
-                </span>
-              </PopOver>
+                {tags.slice(LIST_SIZE).length > 0 && (
+                  <PopOver
+                    html={
+                      <>
+                        {tags.slice(LIST_SIZE).map((tag, index) => (
+                          <Tags
+                            className="tw-bg-gray-200 tw-px-2"
+                            key={index}
+                            tag={`#${tag.tagFQN}`}
+                          />
+                        ))}
+                      </>
+                    }
+                    position="bottom"
+                    theme="light"
+                    trigger="click">
+                    <span className="tw-cursor-pointer tw-text-xs link-text v-align-sub tw--ml-1">
+                      •••
+                    </span>
+                  </PopOver>
+                )}
+              </>
             )}
           </>
         )}
+        {isTagEditable && (
+          <div onClick={() => setIsEditable(true)}>
+            <TagsContainer
+              editable={isEditable}
+              selectedTags={[
+                ...tags.map((tag) => ({
+                  tagFQN: tag.tagFQN,
+                  isRemovable: true,
+                })),
+                { tagFQN: tier, isRemovable: false },
+              ]}
+              showTags={!isTagEditable}
+              tagList={tagList}
+              onCancel={() => {
+                handleTagSelection();
+              }}
+              onSelectionChange={(tags) => {
+                handleTagSelection(tags);
+              }}>
+              {tags.length || tier ? (
+                <button className=" tw-ml-1 focus:tw-outline-none">
+                  <SVGIcons
+                    alt="edit"
+                    icon="icon-edit"
+                    title="Edit"
+                    width="12px"
+                  />
+                </button>
+              ) : (
+                <span className="">
+                  <Tags
+                    className="tw-border-main tw-text-primary"
+                    tag="+ Add new tag"
```

Comment:
Changes `Add new tag` to `Add tag` as it gives wrong impression of being able to add custom tags on the fly

Reply:
Make this change everywhere, either in this PR or make a separate PR for it